src/components/CodeEditor.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <monaco-editor
+    v-model="codeModel"
+    class="code-editor"
+    :lang="lang"
+    :options="{
+      lineHeight: 20,
+      fontFamily: 'Roboto Mono',
+      fontSize: 16,
```

Comment:
```suggestion
      fontSize: 14,
```

16px feels too big for me, especially with large files. I would go with 14.